vru_use_case_scripts/README.md

```diff
@@ -84,3 +84,24 @@ Plot the data:
 ### Example output
 
 ![](docs/example_plot.png)
+
+## `plot_time_to_collision`
+
+This script takes in two CSV files containing vehicle and pedestrian odometry generated from rosbags.
+For each vehicle odometry point, the script propagates the vehicle forward along the path while keeping
```

Comment:
May want to include an assumptions we make about vehicle (point mass or radius) and about stopped vehicles and pedestrians.